test/test_jit.py

```diff
@@ -13087,8 +13062,8 @@ def fn(x):
         self.checkScript(fn, ("y"))
 
         def index_str_to_tensor(s):
-            # type: (str) -> int
```

Comment:
My code puts stricter requirements on type annotations matching the actual return type, so this test fails due to the incorrect type annotation. I think that the `-> int` was just a typo due to 1) the name of the test (`index_str_to_tensor`) and 2) the fact that we're just calling `checkScript` on it (i.e. it doesn't look like the author expected it to fail).

Reply:
I added stricter type annotation requirements in IR Emitter, which causes this test to fail. The test passes if I change the annotation from `-> int` to `Tensor`. I think that the original annotation was just a mistake for two reasons: 1) the name of the test is `index_str_to_tensor`, which makes it sound like the return type should be a Tensor, and 2) the test is only checked using `checkScript` instead of something that would imply an error is expected (e.g. `assertRaisesRegex`)